src/libraries/System.Drawing.Common/src/System/Drawing/Design/CategoryNameCollection.cs

```diff
@@ -26,7 +26,7 @@ public sealed class CategoryNameCollection : ReadOnlyCollectionBase
         /// <summary>
         /// Represents the entry at the specified index of the <see cref='string'/>.
         /// </summary>
-        public string this[int index] => ((string)(InnerList[index]));
+        public string this[int index] => ((string)(InnerList[index])!);
```

Comment:
Should Contains and IndexOf accept nullable?

Also, I don't see anything from preventing the list from having null values.

Reply:
The two points are related. I couldn't find any evidence in either source or tests that the class needs to support nullable strings. In fact, this class is orphaned, it isn't being used by anything else in the assembly.

My personal bias in such cases is to go for non-nullability.

Reply:
Sounds good. Let's leave as is then.